easybuild/easyconfigs/b/Boost/Boost-1.66.0-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,28 @@
+name = 'Boost'
+version = '1.66.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+patches = ['Boost-1.65.1_fix-Python3.patch']
+checksums = [
+    'bd0df411efd9a585e5a2212275f8762079fed8842264954675a4fddc46cfcf60', # boost_1_66_0.tar.gz
+    '5585f98fb67425ec465dcbe2878af046ccd2729a8cdb802acf5d71cfa4022c26', # Boost-1.65.1_fix-Python3.patch
```

Comment:
@RvDijk Style nitpicking (the bot will complain about this too): inline comments should be preceded by a double space.

When adding checksums, it's easier to use `eb --inject-checksums`, which will take care of this stuff for you.